charts/gitlab-ci-pipelines-exporter/templates/_helpers.tpl

```diff
@@ -29,4 +29,15 @@ Create chart name and version as used by the chart label.
 */}}
 {{- define "app.chart" -}}
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
-{{- end -}}
\ No newline at end of file
+{{- end -}}
+
+{{/*
+Create the name of the service account
+*/}}
+{{- define "exporter.serviceAccountName" -}}
+{{- if .Values.serviceAccount.create -}}
```

Comment:
this value is not set / documented in values.yml 🤔

Reply:
My bad. I maintained two versions of values.yaml. One for test cluster, and one for standard repo. So I missed this part. Thanks for reminding me.

Reply:
ah ok, no worries thanks!